platforms/shared/configuration/roles/setup/certmanager/tasks/create_mapping.yaml

```diff
@@ -0,0 +1,45 @@
+---
```

Comment:
The resources created via this file are not idempotent.
Wherever we are supposed to do any change to the current state of the the environment (in this file, the k8s resources), we should first check the current state and perform operations only when the desired state is not same as the current state.
Thus, all the tasks should have a precheck of the existence of the resource and the resource should only be deployed if the pre-check finds the absence of the resource.

Reply:
added the code to check for resources prior to installation